# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/fake/sonypid/makefile-install.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2003-03-25 10:29:49.000000000 +0100
+++ ./Makefile	2003-07-13 14:43:55.000000000 +0200
@@ -4,6 +4,9 @@
 LDFLAGS =
 LIBS = -L/usr/X11R6/lib -lX11 -lXtst
 
+PREFIX = /usr
+DESTBIN = $(PREFIX)/sbin
+
 all: sonypid
 
 sonypid: sonypid.o
@@ -16,3 +19,10 @@
 clean:
 	rm -f sonypid.o sonypid *~
 	rm -f sonypi.h
+
+install:
+	install -m 755 -d $(DESTBIN)
+	install -m 755 sonypid $(DESTBIN)
+
+uninstall:
+	rm -f $(DESTBIN)/sonypid
